2-copy-of-code/lesson-13/scripts/amazon.js

```diff
@@ -72,14 +72,14 @@ document.querySelectorAll('.js-add-to-cart')
       const quantitySelector = document.querySelector(
         `.js-quantity-selector-${productId}`
       );
-      const quantity = quantitySelector.value;
+      const quantity = Number(quantitySelector.value);
 
       if (matchingItem) {
-        matchingItem.quantity += 1;
+        matchingItem.quantity += quantity;
       } else {
         cart.push({
           productId: productId,
-          quantity: 1
+          quantity: quantity
```

Comment:
instead productId: productId and quantity: quantity, use shortHand property 

productId,
quantity